besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1190,6 +1187,17 @@ public void run() {
 
     try {
       configureLogging(true);
+
+      // Set the goquorum compatibility mode based on the genesis file
+      if (genesisFile != null) {
+        genesisConfigOptions = readGenesisConfigOptions();
+        if (genesisConfigOptions.isQuorum()) {
+          // this static flag is read by the RLP decoder
+          GoQuorumOptions.goQuorumCompatibilityMode = true;
+          isGoQuorumCompatibilityMode = true;
+        }
+      }
+
```

Comment:
there's no `--network` option that would be GoQuorum mode, right?

Reply:
right